example/README.md

```diff
@@ -55,9 +55,9 @@ Now users could send inference request with server certificates (`server.crt`).
 
 There are prebuilt docker images could be used for the examples, either in the following docker way or [`kubernates`](./kubernetes/) way. Users could pull them directly and try the example.
 ```
-docker pull occlum/init_ra_server:0.29.0
-docker pull occlum/tf_demo:0.29.0
-docker pull occlum/tf_demo_client:0.29.0
+docker pull occlum/init_ra_server:0.29.1
```

Comment:
Should we change the 0.29.0/0.29.1 to [Occlum latest version] and explain what is the [Occlum latest version] (user could get it from the GitHub main page)? If we do it, then we do not need to update the README for each release.
![image](https://user-images.githubusercontent.com/59240482/197919039-0aef793c-1742-4f38-8c81-e239ff4a3d48.png)

Reply:
@guzongmin Yes, that is the plan once the k8s/docker tf_serving example is added to CI. Before that, it is more safe to 
 provide the verified version to users.